mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -70,7 +149,27 @@ AUI.add(
 							todoList.append(taskHtml);
 
 							taskInput.val('');
+
+							instance._countTasks();
 						}
+					},
+
+					_countTasks: function() {
```

Comment:
The function does more then counts tasks, it also updates UI. it should be named something like "_updateTaskCounts".